org.eclipse.cpacep.ui/src/org/eclipse/internal/cpacep/view/StatisticAction.java

```diff
@@ -33,10 +35,12 @@ public void updateEnablementState() {
     @Override
     public void run() {
 	CpacepView cpacepView = CpacepView.getViewInstance();
+	List<org.eclipse.cpacep.util.Statistics> stats=cpacepView.getCPACEPConncetor().getStatistics();
```

Comment:
How about another name for these Statistics, to avoid the conflict?

Reply:
Hi Thomas,
Changed the name. Could you please merge this with master if there is no issue?. I have completed report generation too. Need some classes(Messages) from this branch.